openfang_config/upstream-overrides/crates/openfang-api/static/js/pages/chat.js

```
@@ -0,0 +1,1274 @@
+        case 'command_result':
+          // Update context pressure if included in command result
+          if (data.context_pressure) {
+            this.contextPressure = data.context_pressure;
+          }
+          this.messages.push({ id: ++msgId, role: 'system', text: data.message || 'Command executed.', meta: '', tools: [] });
+          this.scrollToBottom();
+          break;
+
+        case 'canvas':
+          // Agent presented an interactive canvas — render it in an iframe sandbox
+          let canvasHtml = '<div class="canvas-panel" style="border:1px solid var(--border);border-radius:8px;margin:8px 0;overflow:hidden;">';
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)